src/client.cpp

```diff
@@ -232,71 +239,73 @@ std::string getTaskFileInfo(std::vector<SentFile> files) {
 
 /**
  * @brief Client::sendTaskEncoded
- * @param [in] {TaskType}                 type The type of task
- * @param [in] {std::vector<std::string>} args The task arguments
+ * @param [in] {Scheduler::Task*} task The task arguments
  */
-void Client::sendTaskEncoded(TaskType type, std::vector<std::string> args) {
-    if (type == TaskType::INSTAGRAM) {
-      if (args.size() < 8) {
-        qDebug() << "Not enough arguments to send an IGTask";
-        return;
+void Client::sendTaskEncoded(Scheduler::Task* task) {
+  if (task->getType() == Scheduler::TaskType::INSTAGRAM) {
+    const auto hashtags = KString{std::get<Scheduler::VariantIndex::QSTRING>(task->getTaskArgument("hashtags"))};
```

Comment:
Will clean this up. The issue here is that FlatBuffers expects an implementation of an interface which includes c_str(), to get the C string (like an std::string). I was really hoping to switch over to only using QString, so we could get the niceties it has to offer (better syntax for string comparison, for example). Unfortunately, it's missing that interface, so I'm copying the QStrings into a new KString method, which is just a QString with that extra method.

Solution 1. Go back to std::string
Solution 2. Implement enough of QString on KString so we can switch to it, throughout the project. (Mostly, we can defer to parent class)